surveyor/surveyor.go

```diff
@@ -108,6 +110,8 @@ func connect(opts *Options) (*nats.Conn, error) {
 	nopts := []nats.Option{nats.Name(opts.Name)}
 	if opts.Credentials != "" {
 		nopts = append(nopts, nats.UserCredentials(opts.Credentials))
+	} else if opts.NATSUser != "" {
```

Comment:
should we check if password is also set? warn if both isnt set maybe? Guess worst what will happen here is a error will get logged so probably isnt really needed but might be a bit friendlier experience

Reply:
Good thought... however, if there's no password, the username is assumed to be a token by NATS.  Not commonly used but still supported.  I'll update the parameter description.

Reply:
Ah, didnt know that! Looks good then